open-api/clients/javascript-client/src/api/CommunityApi.js

```diff
@@ -0,0 +1,282 @@
+/**
+ * Havfen Community-API
+ * Hafven Community API [Development]. This might link to the homepage [http://hafven-community.ngrok.io](http://hafven-community.ngrok.io). Notice that all html tags are valid in this description. Feel free to be descriptive.
+ *
+ * OpenAPI spec version: 0.0.1
+ * Contact: apiteam@hafven-community.ngrok.io
+ *
+ * NOTE: This class is auto generated by the swagger code generator program.
```

Comment:
is there any option generation this class as part of CI instead of using git for storing generated code?

Reply:
@terribleherbst Great idea! Two ways:

For a nice tutorial https://blog.cloudboost.io/adding-swagger-to-existing-node-js-project-92a6624b855b


a) "on-the-fly solutions": no code-generation is required. There many npm modules helping there. Say https://github.com/swagger-api/swagger-node, or likely better maintained https://www.npmjs.com/package/swagger-ui-express. 


b) Do the codegen in the circleCI environment, say using the original package, https://www.npmjs.com/package/swagger-node-codegen, or in a 'ready-to-use' docker hub swagger codegen image.

Reply:
Feel free to add a Readme.md under open-api ;)